book/06-github/sections/1-setting-up-account.asc

```diff
@@ -1,86 +1,86 @@
-=== Account Setup and Configuration
+=== Pengaturan dan Konfigurasi Akun
 
-(((GitHub, user accounts)))
-The first thing you need to do is set up a free user account.
-Simply visit https://github.com[], choose a user name that isn't already taken, provide an email address and a password, and click the big green ``Sign up for GitHub'' button.
+(((GitHub, akun pengguna)))
+Hal pertama yang perlu Anda lakukan adalah membuat akun pengguna gratis.
+Hanya mengunjungi https://github.com[], pilih nama pengguna yang belum diambil, sediakan alamat surel dan kata sandi, dan klik tombol hijau yang besar 'Sign up for GitHub'.
```

Comment:
5. Cukup dengan mengunjungi .... atau Cukup kunjungi ....

Tanda petiknya tidak usah diubah untuk Sign up for Github, biarkan saja seperti itu.